manta-js/package/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "manta.js",
+  "version": "1.0.0",
+  "description": "manta.js sdk",
+  "main": "sdk.ts",
+  "dependencies": {
```

Comment:
Might want to add a build script that deletes the current build and then runs `tsc`